cdap-docs/developers-manual/source/conf.py

```diff
@@ -9,6 +9,28 @@
 
 # Setup the config
 sys.path.insert(0, os.path.abspath('../../_common'))
+from common_conf import setup as _setup
 from common_conf import *
 
 html_short_title_toc, html_short_title, html_context = set_conf_for_manual()
+
+def setup(app):
+    # Call imported setup
+    _setup(app)
+    # Add a custom config value that can be used for conditional content
+    app.add_config_value('snapshot_version', '', 'env')
+
+# Set the condition
+if release.endswith('SNAPSHOT'):
+    snapshot_version = True
+    archetype_repository_version = ('-DarchetypeRepository=https://oss.sonatype.org/content/repositories/snapshots/' 
+      + " -DarchetypeVersion=%s" % release)
+else:
+    snapshot_version = False
+    archetype_repository_version = "-DarchetypeVersion=%s" % release
+    
+rst_epilog += """
+
+.. |archetype-repository-version| replace:: %(archetype_repository_version)s
```

Comment:
Yes. See the comment at the top of the PR. An earlier version of this was run against a SNAPSHOT when the release/4.0 was accidentally changed; the results can be see at the two different _Page of interest_.